statsmodels/sandbox/regression/penalized.py

```diff
@@ -76,13 +76,13 @@ class TheilGLS(GLS):
 
     The parameter estimates solves the moment equation:
 
-    .. math:: (X' \\Sigma X + \\lambda R' \\sigma^2 \\Simga_p^{-1} R) b = X' \\Sigma y + \\lambda R' \\Simga_p^{-1} q
```

Comment:
this is already properly escaped

Reply:
I thought adding the “r” to the docstring means you only use one backslash.  Am I confusing this with something else?

Reply:
r means no escaping, without r backslash needs escaping \->\\.

Reply:
I would use r and one \ since the docstring is mixed (some \ some \\).

Reply:
@bashtage I thought that’s what I did here.  Did I miss something?

Reply:
You are right, The docstring was mixed so you could have added r and moved to single escapes or left the r off and moved everything to double escape. I was mostly addressing @josef-pkt point about the escaping being correct which was right for that line but wrong for the docstring overall.